lib/bridge/github/contents.rb

```diff
@@ -23,7 +23,7 @@ def issue_template_content
         content = nil
       end
 
-      return content
+      return content ? content.force_encoding("utf-8") : content
```

Comment:
Its ensuring explicit `nil`, though its almost certain that a non-true content value would return `nil` anyways I prefer explicit over implicit in this kind of situation.